### PR TITLE
Ticket Tasks Enhancement

### PR DESCRIPTION
**Pull request by tsmr**
An old ticket :P https://github.com/glpi-project/glpi/issues/1159

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #1159

image

& hover

total-hover

### CODE REVIEW

File: inc/commonitilobject.class.php [hunk 1]
```diff
@@ -6657,7 +6657,71 @@ function viewEditSubitem" . $this->fields['id'] . "$rand(e, itemtype, items_id,
 
       echo "</ul>"; // timeline_choices
       echo "<div class='clear'>&nbsp;</div>";
-
+      //total_actiontime stat
+      if (Session::getCurrentInterface() != 'helpdesk') {
+         echo "<div class='timeline_stats'>";
+         echo "<h3>";
+         $total_actiontime = 0;
+         foreach ($DB->request("glpi_tickettasks",
```
Comment (trasher) [inline inc/commonitilobject.class.php:6665]: Do not call `request` inside of a loop; store the restul in a variable before and use that variable in the loop (for all loops).

Comment (trasher) [reply inc/commonitilobject.class.php:6665]: Also, do **not** use raw SQL.

Comment (tsmr) [reply inc/commonitilobject.class.php]: Better ?

Comment (trasher) [reply inc/commonitilobject.class.php]: :+1:

**Pull request merged**